src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -231,8 +231,8 @@ public void Exit()
 
 		public static void Start(global::Windows.UI.Xaml.ApplicationInitializationCallback callback)
 		{
-			ApplicationLanguages.ApplyCulture();
 			StartPartial(callback);
```

Comment:
About the ordering change:

`StartPartial` is what creates an `Application` instance. So we want that to run first so that we can find the EntryAssembly set in Application's constructor.